dotcms-integration/src/test/java/com/dotmarketing/quartz/job/PruneTimeMachineBackupJobTest.java

```diff
@@ -0,0 +1,163 @@
+package com.dotmarketing.quartz.job;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.FileUtil;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Collection;
+import java.util.stream.Collectors;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+public class PruneTimeMachineBackupJobTest {
+    final PruneTimeMachineBackupJob pruneTimeMachineJob = new PruneTimeMachineBackupJob();
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    private File createTimeMachineFolder(final File parentFolder, final String fileName,
+                                         final int howLongWasModified) throws IOException {
+        final Calendar expireDate = Calendar.getInstance();
```

Comment:
Shouldn't we use `java.time` classes for date-time operations?

Reply:
I am going to fix it on the next PR, because this PR is already on the merge queue